src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -134,6 +139,45 @@ public static IEnumerable<LowLevelClientMethod> BuildMethods(RestClientBuilder b
             }
         }
 
+        private LowLevelConvenienceMethod BuildConvenienceMethod(in LowLevelClientMethod clientMethod, RestClientBuilder builder)
+        {
+            List<Parameter> parameters = new List<Parameter>();
+            MethodSignature protocolSignature = clientMethod.Signature;
+            InputOperation operation = clientMethod.RequestMethod.Operation;
+            Parameter? bodyParameter = null;
+            foreach (var parameter in protocolSignature.Parameters)
+            {
+                if (parameter == KnownParameters.RequestContent || parameter == KnownParameters.RequestContentNullable)
+                {
+                    bodyParameter = builder.GetBodyParameter(operation, DefaultNamespace);
```

Comment:
See my comment at the top. Here you can change to:
```suggestion
                    bodyParameter = operation.GetBodyParameter(DefaultNamespace);
```

Reply:
Same reason above. I cannot pass the `_typeFactory` to `InputOperation`.